test/pytest/test_device_diagnostics.py

```
@@ -12,16 +12,12 @@ def test_device_list_logs(harness):
     ret = GetDiagnosticFilesList(DiagnosticsFileList.LOGS).run(harness)
 
     if not ret.files:
-        log.info("No log files")
-        assert True
+        log.warning("No log files")
```

there was assert here, maybe raise an exception? similarly below.